playbook/runner/runner.go

```diff
@@ -142,12 +142,26 @@ func CheckDelay(ctx context.Context, playbook models.Playbook, run models.Playbo
 	return false, nil
 }
 
+func getEligibleAgents(spec v1.PlaybookSpec, action *v1.PlaybookAction, run models.PlaybookRun) []string {
+	if run.Parameters != nil && run.Parameters["agent_id"] != "" {
```

Comment:
Where does this agent_id come from? RunParams has a AgentID field which maps to run.AgentId = we can't inject it into parameters, as then they are no longer valid (extra param)

Reply:
ah I misread `RunParams.AgentID` to be a run parameter.